duckduckgo/iOS
# long press preview setting

Task/Issue URL: https://app.asana.com/0/414235014887631/1145895736139511
Tech Design URL: 
CC:

**Description**:

Allows user to disable long press previews on links (not images since that's iOS)

**Steps to test this PR**:
1. Visit a site with links, preview should be shown for long press
1. Go to settings and change toggle for "Long-Press Links" to off
1. Visit a site with links, preview should not be shown for long press
1. Go to settings and change toggle for "Long-Press Links" to on
1. Visit a site with links, preview should be shown for long press
1. Ensure pixels fire for the different states

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
